### ISSUE TITLE
`getRuleInfo` not working

### ISSUE THREAD

**Issue by fisker** [not-an-aardvark/eslint-plugin-eslint-plugin#106]
[`getRuleInfo`](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/02c1f347f2f0881e364724a4fce69e9da2ce9313/lib/utils.js#L87) seems not working if rule write in this way

```js
'use strict';

const create = () => {};

module.exports = {create};
```

See https://github.com/sindresorhus/eslint-plugin-unicorn/blob/e7e25b82526e42970043c39a512cbdfb889b7cb7/rules/better-regex.js#L107

I dig a little bit, [this line](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/02c1f347f2f0881e364724a4fce69e9da2ce9313/lib/utils.js#L32) , `scopeManager.acquire(node)` is null, so it will use `globalScope`. If I change to `scopeManager.globalScope.childScopes[0]` it will work.

**Comment by aladdin-add** [comment]
PRs are welcome! 😄

**Comment by aladdin-add** [comment]
released in eslint-plugin-eslint-plugin@3.0.1
